app/controllers/queue_items_controller.rb

```diff
@@ -27,4 +39,13 @@ def next_list_item
   def queue_includes?(video)
     current_user.queue_items.map(&:video).include?(video)
   end
+
+  def update_queue_items
+    ActiveRecord::Base.transaction do
+      params[:queue_items].each do |item|
+        queue_item = QueueItem.find(item["id"])
+        queue_item.update_attributes!(list_order: item["list_order"], video_rating: item["rating"]) if queue_item.user == current_user
```

Comment:
This line http://cl.ly/image/2g3O2m1y2C04 is over 80 characters - to keep lines readable and to make debugging easier please refactor any lines over this limit. You can see this referenced in the style guide here - https://github.com/bbatsov/ruby-style-guide#80-character-limits